src/status_im/navigation.cljs

```diff
@@ -38,15 +38,16 @@
     (assoc-in [:navigation/screen-params view] screen-params)))
 
 (fx/defn navigate-to-cofx
-  [{:keys [db]} go-to-view-id screen-params]
+  [{:keys [db] :as cofx} go-to-view-id screen-params]
   {:db
    (-> (assoc db :view-id go-to-view-id)
        (all-screens-params go-to-view-id screen-params))
-   ::navigate-to [go-to-view-id screen-params]})
+   ::navigate-to [go-to-view-id screen-params]
+   ;; simulate a navigate-to event so it can be captured be anon-metrics
+   ::anon-metrics/transform-and-log {:coeffects {:event [:navigate-to go-to-view-id screen-params]}}})
```

Comment:
whats that ?

Reply:
We wrote an interceptor to keep the metrics code away from app code. But some events are using `fx/merge`, not via direct dispatch. Also sometimes, we might want to capture events in an `fx/merge` chain. 

The effect `::transform-and-log` and the handler `::hoax-event-capture` help out with this situation. Both call `transform-and-log` function internally, which expects an Re-frame context (so it can parse the event and strip unnecessary information).

Since this event-handler is used with `fx/merge`, we cannot intercept it. So need to use the effect directly.